dts/arm/nxp/nxp_lpc55S3x_common.dtsi

```diff
@@ -225,6 +229,12 @@
 		#size-cells = <0>;
 	};
 
+	css: css@4003000 {
+		compatible = "nxp,lpc-css";
+		reg = <0x4003000 0x200>;
+		status="okay";
```

Comment:
This should be `disabled` and `enabled` in the board that needs it.

Reply:
I specifically did it this way even though I know other devices do it at the board level because for all of our other entropy devices we set status okay at the soc level